data/Sri Guru Granth Sahib Ji/0257.json

```diff
@@ -2242,7 +2242,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "Shall not be turned away.",
+              "translation": "shall not be turned away.",
```

Comment:
Another suggested reversal. Current and suggested below.

Current:
```
ਨੰਨਾਕਾਰੁ; ਨ ਹੋਤਾ ਤਾ ਕਹੁ ॥
nµnwkwru; n hoqw qw khu ]
Those who have been given the Mantra of the Naam by the Guru,

ਨਾਮੁ ਮੰਤ੍ਰੁ; ਗੁਰਿ ਦੀਨੋ ਜਾ ਕਹੁ ॥
nwmu mMqRü; guir dIno jw khu ]
shall not be turned away.
```

Suggested:
```
ਨੰਨਾਕਾਰੁ; ਨ ਹੋਤਾ ਤਾ ਕਹੁ ॥
nµnwkwru; n hoqw qw khu ]
Those shall not be turned away,

ਨਾਮੁ ਮੰਤ੍ਰੁ; ਗੁਰਿ ਦੀਨੋ ਜਾ ਕਹੁ ॥
nwmu mMqRü; guir dIno jw khu ]
who have been given the Mantra of the Naam by the Guru.
```